Dalamud/Interface/UiBuilder.cs

```diff
@@ -100,6 +102,11 @@ internal UiBuilder(string namespaceName)
     /// </summary>
     public event Action HideUi;
 
+    /// <summary>
+    /// Gets the manager for external, WinAPI-based drag and drop functionality.
+    /// </summary>
+    public IDragDropManager DragDropManager => this.dragDropManager;
```

Comment:
It's probably safer for DragDropManager to be a plugin service instead, since the DI will recognize that and handle its lifecycle properly. See e.g. GameConfig or PartyList.